### ISSUE TITLE
LibPD version

### ISSUE THREAD

**Issue by mgsx-dev** [b2renger/PdDroidPublisher#11]
The libPD version is not synchronized with last Pd vanilla version.

For instance, array function are not supported.

The problem is that pd-for-android reference an old version of libPD and libPD reference a strange and maybe old version of PureData (from source forge, not github).

We first have to know which version our libPD is build on. And then see you to easily upgrade all things to keep synchronized with Pure Data releases.

**Comment by mgsx-dev** [comment]
libPD have just upgrade to Pd 0.46, we have to pull the new version and test it.

**Comment by mgsx-dev** [comment]
I've pushed libpd migration in branch https://github.com/b2renger/PdDroidPublisher/tree/migration/libpd for testing if it's resolve #34 . It's highly recommanded to create a new local clone this branch because submodules has been removed, eclipse setup has drastically changed (as per the readme update).

this solution is : 
- we dont use pd-for-android submodule anymore
- we use org.puredata.android:pd-core:1.0.0-rc3 depenedency instead
- this AAR library and its dependencies are converted to eclipse project and then imported in workspace.

good news : we get rif of submodule and use external dependencies
bad news is AAR are well suited for Android Studio but not for eclipse, so eclipse setup is a little tricky for now but we'll reconsider if #34 is fixed with these changes.
